src/storekit.cs

```diff
@@ -763,8 +744,65 @@ enum SKCloudServiceSetupMessageIdentifier {
 		[Field ("SKCloudServiceSetupMessageIdentifierPlayMusic")]
 		PlayMusic,
 	}
+#endif
 
-	[NoWatch, iOS (11,0), TV (11,0)]
+	[Mac (11,0), Watch (7,0), iOS (9,3), TV (9,2)]
+	[BaseType (typeof (NSObject))]
+#if XAMCORE_3_0 // Avoid breaking change in iOS
+	[DisableDefaultCtor]
+#endif
+	interface SKCloudServiceController {
```

Comment:
Any reason to move this class? It makes the diff harder to review :/

Reply:
Moved it out of the #if over adding extra #if #else

Reply:
yeah, better the extra `#if` next time
or do a separate PR to move things
moving and adding makes the diff more difficult to review